tests/js/spec/components/assigneeSelector.spec.jsx

```diff
@@ -28,18 +32,18 @@ describe('AssigneeSelector', function() {
   };
 
   beforeEach(function() {
-    this.sandbox = sinon.sandbox.create();
-    stubReactComponents(this.sandbox, [LoadingIndicator]);
+    sandbox = sinon.sandbox.create();
+    stubReactComponents(sandbox, [LoadingIndicator]);
 
-    this.sandbox.stub(MemberListStore, 'getAll').returns([USER_1, USER_2]);
-    this.sandbox.stub(GroupStore, 'get').returns({
+    sandbox.stub(MemberListStore, 'getAll').returns([USER_1, USER_2]);
+    sandbox.stub(GroupStore, 'get').returns({
       id: 1337,
       assignedTo: null
     });
   });
 
   afterEach(function() {
-    this.sandbox.restore();
+    sandbox.restore();
```

Comment:
Is there not a possibility of the now closure-scoped `sandbox` variable having different values between async test executions?

Reply:
Hmm, don't tests still execute serially regardless if they are async?